draft-irtf-cfrg-frost.md

```diff
@@ -706,10 +713,11 @@ Each signer in round one generates a nonce `nonce = (hiding_nonce, binding_nonce
     return (nonce, comm)
 ~~~
 
-The private output `nonce` from Participant `P_i` is stored locally and kept private
-for use in the second round. This nonce MUST NOT be reused in more than one invocation
-of FROST, and it MUST be generated from a source of secure randomness. The public output
-`comm` from Participant `P_i` is sent to the Coordinator.
+The outputs `nonce` and `comm` from Participant `P_i` should both be stored locally and
+kept for use in the second round. The `nonce` value is secret and MUST NOT be shared, whereas
+the public output `comm` is sent to the Coordinator. The nonce values produced by this
+function MUST NOT be reused in more than one invocation of FROST, and it MUST be generated
+from a source of secure randomness.
 
```

Comment:
The commitments should also not be reused, right?

Reply:
Right, but that's implicit from the requirement that nonces must not be reused, so I don't think we need to explicitly say it.